test/indexTest.js

```diff
@@ -1,23 +1,112 @@
+describe("index.js", () => {
+  describe("defines saturdayFun function declaration as specified", function() {
```

Comment:
in the readme 'method' is used for `saturdayFun` and `mondayWork`. I think function is better